src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -36,6 +78,9 @@ var AppComponent = React.createClass({
             {model.formatTasksRunning()}
           </span> / {model.get("instances")}
         </td>
+        <td className="text-center health-bar-column">
```

Comment:
Here's an image showing the health bar under `Status`:
![](https://s3.amazonaws.com/f.cl.ly/items/0g1V230s3n3T0M1x1S2W/Image%202015-01-19%20at%2019%3A16%3A23.png)